hpOneView/resources/servers/logical_enclosures.py

```diff
@@ -112,159 +71,83 @@ def get_all(self, start=0, count=-1, filter='', sort='', scope_uris=''):
         Returns:
             list: A list of logical enclosures.
         """
-        return self._client.get_all(start, count, filter=filter, sort=sort, scope_uris=scope_uris)
-
-    def get_by(self, field, value):
-        """
-        Gets all logical enclosures that match the filter.
-
-        The search is case-insensitive.
-
-        Args:
-            field: Field name to filter.
-            value: Value to filter.
-
-        Returns:
-            list: A list of logical enclosures.
-        """
-        return self._client.get_by(field, value)
+        return self._helper.get_all(start, count, filter=filter,
```

Comment:
i was assuming it would call this method by default. If not, then we have to overwrite it in ServerHardware resource as well because we can filter server hardware with scopeUris similar to this.

Reply:
Server hardware doesn't have scopeUri query param.